src/i18n/en.js

```diff
@@ -211,6 +211,8 @@ by Doppler and how many don't. By tracking the user's journey you'll be able to
     delivery_rate: `Delivery Rate`,
     emails_delivered: `Emails Delivered:`,
     hard_and_soft: `Hard and Soft bounces`,
+    header_description_shipped: `This is a report of your sent campaign.`,
```

Comment:
How about this? @mrunge86 what do you think?
```suggestion
    header_description_shipped: `This is a Report of your Sent Campaign.`,
```